### DISCUSSION TITLE
Duration computation

### DISCUSSION THREAD

**Discussion by ringostarlet** [hklages/node-red-contrib-sonos-plus#260] [Q&A] [answered]
Hi,

I have the flow going and its really great. Unfortunately the duration of my polly mp3 is always the same resulting in a pause or cutting off the text if its too long. How can I compute the duration or have Sonos-plus obeying the duration of my mp3?

Thanks

**Comment by hklages** [comment]
Hi.

What command do you use: group.play.notification? Are your players grouped or is it a single player?

**Comment by ringostarlet** [comment]
Hi,

yes, I do use group.play.notification. I have a group (One and Five) set up, yes.

**Comment by hklages** [comment, marked as answer]
Can you send me an example mp3 file - one that does not play in a correct way? Either here or attached to an email to nrcsplus@gmail.com

If you have only 2 player you can try the brand new SONOS build in function. Just send the mp3 file to both players with command player.play.clip. Thats new in 6.7.3

**Comment by ringostarlet** [comment]
I tried the player.play.clip command and this works. I can play on both players at the same time (slightly echoing).

I mailed you a sample file as I cannot upload mp3 files here. Thanks for your help!

**Comment by hklages** [comment]
>  tried the player.play.clip command and this works. I can play on both players at the same time (slightly echoing).

Yes, there may be an echo. You could reduce that by sending the data parallel. 
![parallel](https://user-images.githubusercontent.com/17273119/230121161-98db4f9b-8356-4ed0-9abf-e17d4814affe.png)



If that is fine for you - use it. That is a SONOS build function and maybe in the future SONOS will support groups.

**Comment by ringostarlet** [comment]
Yes, I did it in parallel, still a slight echo but good enough. Thanks
